installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+readonly DOCKERHUB_USER="${0}"
+readonly DOCKERHUB_PASS="${1}"
+readonly DOCKERHUB_ORG="ansibleplaybookbundle"
+
+curl -s https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml > /tmp/deploy-ansible-service-broker.template.yaml
```

Comment:
This was only updated 4 days ago so it may be worth leaving this until we must bring it down in case it gets any more updates. I'll make an issue for it.